Makefile

```diff
@@ -239,8 +239,8 @@ release/gen-templates:
 # operator version is changed. The results are committed to Git.
 .PHONY: release/gen-files
 release/gen-files: release/gen-templates
-	$(MAKE) CHANNEL=beta IS_DEFAULT_CHANNEL=0 release/update-pkg-manifest && \
-	$(MAKE) CHANNEL=beta IS_DEFAULT_CHANNEL=0 release/opm-build-bundle && \
+	$(MAKE) release/update-pkg-manifest && \
```

Comment:
What variables are we using now?

Reply:
I will update the release doc to use `make releaase/gen-files CHANNEL=stable IS_DEFAULT_CHANNEL=1` for the 2.0.x release.

Reply:
Looks like I need to export these variables, because we recursively call `$(MAKE)`.

Reply:
Actually it works fine even without exporting these variables. All good.